### ISSUE TITLE
Focus on iOS and OSX

### ISSUE THREAD

**Issue by ghost** [takeoutweight/clojure-scheme#5]
I think you should call it iClojure or something and focus on iOS and OSX. 
Maybe it's just me but I was a surprised to hear that clojure-scheme is good for writing iPhone apps. Not so many people know scheme, but a lot of people want to write native iOS/OSX applications in clojure.

**Comment by takeoutweight** [comment]
I have thought about working on a really nice tool-chair using CMake to build XCode projects. Perhaps branding this workflow stuff could make the iOS connection more explicit

**Comment by ghost** [comment]
Yes! That's even better. It would really boost your project. Man! Clojure native on iOS/OSX. That's amazing!
